releases-beta/how-to-guides/create-custom-environment-variables.md

```diff
@@ -0,0 +1,56 @@
+# Create Custom Environment Variables
+
+Custom environment variables allow you to define and manage key-value pairs that can be accessed during your software build and deployment process. This ensures that your release pipeline can use specific configurations for different environments.
+
+## Generate Custom Variables in Your Project Config
```

Comment:
Besides "Project Config", users can also configure workflow params for each environments. You can click into settings of each environment to take a look. The difference is that we also provide an Aviator variable "Environment Name" which is not available for a project